environment/src/main/java/com/sequenceiq/environment/environment/sync/EnvironmentJobService.java

```diff
@@ -0,0 +1,42 @@
+package com.sequenceiq.environment.environment.sync;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.environment.environment.domain.Environment;
+import com.sequenceiq.statuschecker.service.JobService;
+
+@Component
+public class EnvironmentJobService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(EnvironmentJobService.class);
+
+    @Inject
+    private AutoSyncConfig autoSyncConfig;
+
+    @Inject
+    private JobService jobService;
+
+    public void deleteAll() {
+        jobService.deleteAll();
+    }
+
+    public void schedule(Environment environment) {
+        if (autoSyncConfig.isEnabled()) {
+            jobService.schedule(new EnvironmentJobAdapter(environment));
+            LOGGER.info("{} is scheduled for auto sync", environment.getName());
+        }
+    }
+
+    public void unschedule(Long envId) {
+        jobService.unschedule(envId.toString());
+    }
+
+    public void unschedule(Environment environment) {
+        jobService.unschedule(environment.getId().toString());
```

Comment:
call this class' unschedule(Long) and move logging there

Reply:
I cannot log there, because here the log message is writing the name. The unschedule(long) will be called here instead of jobService